### PR TITLE
Bubbl.us

### PR DESCRIPTION
**Pull request by Rami666**
i did the requested changes, even deleted the Dating App section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,11 @@ See the [contributing guide](CONTRIBUTING.md) for detailed instructions on how t
 
 ### Knowledge Management
 
+#### [Bubbl.us](https://bubbl.us/) - **Bubbl.us** Is a web-based mind mapping app allows you to build, save, and share up to three mind maps on the free plan.
```
Comment (OliveiraDCI) [inline README.md:35]: That looks good, I would just suggest changing the title#4 format with the #### to a simple '-' following the same structure format as the rest of the document.

**Review by OliveiraDCI — CHANGES_REQUESTED** [review]
Please change the title format to bullet point format, as mentioned in the file comment.

**Review by OliveiraDCI — APPROVED** [review]
Lgtm. 
@HelonDCI please merge.

**Review by OliveiraDCI — APPROVED** [review]
Lgtm.
@HelonDCI  please merge.

**Comment by OliveiraDCI** [comment]
It is already reviewed and approved, we just need to wait for Helon to merge because he is the only one with the rights to do that.

**Review by HelonDCI — APPROVED** [review]

**Pull request merged**